landscape/client/broker/tests/public.ssl

```diff
@@ -1,9 +1,9 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
I think you already fixed this and need to rebase?

Reply:
Right, gone by merging `master`.